scripts/regolith-sway-clamshell

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+ACTION=$( trawlcat regolith.clamshell.action "DISABLE_OUTPUT" )
```

Comment:
Given that this key controls window manager behavior, a key name prefix of `wm.` may be more consistent.